server/sonar-web/src/main/js/components/SourceViewer/SourceViewerCode.tsx

```diff
@@ -165,26 +169,25 @@ export default class SourceViewerCode extends React.PureComponent {
         highlightedLocationMessage={optimizedLocationMessage}
         highlightedSymbols={optimizedHighlightedSymbols}
         issueLocations={this.getIssueLocationsForLine(line)}
+        issuePopup={this.props.issuePopup}
         issues={issuesForLine}
         key={line.line}
         last={index === this.props.sources.length - 1 && !this.props.hasSourcesAfter}
         line={line}
+        linePopup={this.props.linePopup}
         loadDuplications={this.props.loadDuplications}
-        onClick={this.props.onLineClick}
-        onCoverageClick={this.props.onCoverageClick}
-        onDuplicationClick={this.props.onDuplicationClick}
         onIssueChange={this.props.onIssueChange}
+        onIssuePopupToggle={this.props.onIssuePopupToggle}
         onIssueSelect={this.props.onIssueSelect}
         onIssueUnselect={this.props.onIssueUnselect}
-        onIssuesOpen={this.props.onIssuesOpen}
         onIssuesClose={this.props.onIssuesClose}
+        onIssuesOpen={this.props.onIssuesOpen}
+        onLinePopupToggle={this.props.onLinePopupToggle}
         onLocationSelect={this.props.onLocationSelect}
-        onSCMClick={this.props.onSCMClick}
         onSymbolClick={this.props.onSymbolClick}
         openIssues={this.props.openIssuesByLine[line.line] || false}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the unnecessary boolean literal. ([typescript:S1125](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1125))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qZxw6XGHKvPuQTN&open=AWHb6qZxw6XGHKvPuQTN)

[ISSUE_KEY]: AWHb6qZxw6XGHKvPuQTN